python/lsst/ip/isr/isrTask.py

```diff
@@ -188,6 +188,11 @@ class IsrTaskConfig(pexConfig.Config):
         default=True,
         doc="Assemble amp-level exposures into a ccd-level exposure?"
     )
+    expectWcs = pexConfig.Field(
+        dtype=bool,
+        default=True,
+        doc="Input science images are expected to have a WCS (not true for e.g. spectrographs)"
```

Comment:
I'd phrase this "Expect input science images to have a WCS (set False for e.g. spectrographs)".